doc/requirements.txt

```diff
@@ -1,3 +1,4 @@
+furo
```

Comment:
Maybe one should actually change the script to use `doc/requirements.txt`? But then, I believe that the `release/requirements.txt` may be used for the release, so probably both files should be in sync (i.e., only one file...).